adonis-typings/i18n.ts

```diff
@@ -237,6 +237,11 @@ declare module '@ioc:Adonis/Addons/I18n' {
      */
     formatMessage(identifier: string, data?: Record<string, any>, fallbackMessage?: string): string
 
+    /**
+     * Short usage of formatMessage
```

Comment:
The description should be `Shorthand method for formatUsage`. Also it will be nice to drop in the `@alias` directive.

```ts
/**
 * Shorthand method for formatUsage
 * @alias formatUsage
 */
```

Reply:
Fixed